app/src/main/java/com/shagalalab/constitution/chapter/adapter/ChapterAdapter.kt

```diff
@@ -0,0 +1,31 @@
+package com.shagalalab.constitution.chapter.adapter
+
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import androidx.recyclerview.widget.RecyclerView
+import com.shagalalab.constitution.R
+import com.shagalalab.constitution.data.models.ChapterModel
+
+class ChapterAdapter(private val itemClickListener: ItemClickListener) :
+    RecyclerView.Adapter<ChapterItemViewHolder>() {
+
+    private var models: List<ChapterModel> = arrayListOf()
+
+    fun setData(models: List<ChapterModel>) {
+        this.models = models
+        notifyDataSetChanged()
+    }
+
+    override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): ChapterItemViewHolder {
+        val view = LayoutInflater.from(parent.context).inflate(R.layout.item_view, parent, false)
```

Comment:
Seems we need to fix `item_view`:

![Screenshot_1572043113](https://user-images.githubusercontent.com/8268042/67608620-c1ea5c80-f7d4-11e9-8e4e-aaa22993a7b8.png)

First of all, make TextView's width to match its parent. Second, we should get rid of having a fixed height for parent view, as it is not working well when we have multiline description (there is `minHeight` parameter, probably you can make the height wrap content, but set `minHeight`). Play around with it, basically, we need to have the same padding between top and bottom like in `XXV bap`, so regardless of how many lines in description, keep the top and bottom paddings.